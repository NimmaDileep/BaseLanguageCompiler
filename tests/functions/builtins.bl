struct Car {
    wheels: Int,
    doors: Int
}

// Print
print(3);
print("Hello");
print(false);
print([3, 2]);
print(Car(4, 1));

// Concat
print(concat("Hello", " World"));

// Str
var x: Int = 5;
var y: String = str(x);

print(y);
