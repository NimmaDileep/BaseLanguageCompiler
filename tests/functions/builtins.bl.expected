3
Hello
false
[3, 2]
Car{wheels=4,doors=1}
Hello World
5
